src/packet.rs

```diff
@@ -76,6 +80,12 @@ impl<'a> RtpPacket<'a> {
             });
         }
 
+        // Detect RTCP packets here. We still allow payload types 72-76 as long
+        // as the marker bit is not set.
+        if (200..=204).contains(&data[1]) {
```

Comment:
Not sure if this is sufficient but that's what libgstrtp does. Can transport/payload feedback packets be the first packet in a compound RTCP packet?

Reply:
> Can transport/payload feedback packets be the first packet in a compound RTCP packet?

From memory, yes if reduced sized RTCP is in use. Any RTCP packet can be the first in a RTCP compound packet: https://datatracker.ietf.org/doc/html/rfc5506#section-4.1

See https://www.rfc-editor.org/rfc/rfc5761.html#section-4 for what RTCP/RTP multiplexing should be able to acoomodate.

Reply:
The problem I have with this approach of flat out denying anything that might look like a RTCP packet, is that it is entirely possible that a RTP packet that almost looks like a RTCP packet used in a non rtcp-mux scenario should be able to be parsed just fine.

i.e., this needs knowledge about whether rtcp-mux is the correct scenario before disallowing parsing the particular construction of a RTP packet.

Reply:
Maybe we could have a separate standalone function here `fn RtpPacket::might_be_rtcp(slice: &[u8]) -> bool` and move this problem to the caller?

Reply:
Or alternatively instead of such a function, the caller can check its PT map and if doing rtcp-mux for values not showing up in the PT map in the range 64-95 it would check if they're RTCP packets.

That requires no changes at all here then.